spk/jackett/src/service-setup.sh

```diff
@@ -0,0 +1,34 @@
+
+# Package specific behaviors
+# Sourced script by generic installer and start-stop-status scripts
+
+service_prestart ()
+{
+    # Replace generic service startup
+
+    MONO_PATH="${SYNOPKG_PKGDEST}/../mono/bin"
+    PATH="${SYNOPKG_PKGDEST}/bin:${MONO_PATH}:${PATH}"
+    MONO="${MONO_PATH}/mono"
+    JACKETT="${SYNOPKG_PKGDEST}/share/${SYNOPKG_PKGNAME}/JackettConsole.exe"
+    HOME_DIR="${SYNOPKG_PKGDEST}/var"
+
+    echo "Starting Jackett as user ${EFF_USER}" >> ${LOG_FILE}
+    COMMAND="env HOME=${HOME_DIR} PATH=${PATH} LD_LIBRARY_PATH=${SYNOPKG_PKGDEST}/lib ${MONO} ${JACKETT} --PIDFile ${PID_FILE}"
+
+    if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 6 ]; then
```

Comment:
These if statements are same as generic script.
I think moving all variables settings out of `service_prestart` (so removing this function) and provide `SERVICE_COMMAND` at the begging of this script should work.

Reply:
No that won't work because the process never detaches: no `&` at the end when using SERVICE_COMMAND` for processes that don't support deamon.
So DSM will stay in a constant state of `Starting` since the `start` never returns.
So that's why we need this special function. Or integrated in generic approach :)

Reply:
*LOL* Just came here to write the same! Agree with @Safihre . This is another candidate for the `start-stop-daemon` approach. I propose to keep this package as is for the time being and update once that approach is established...